src/api/integrator/mapbox-gl.js

```diff
@@ -62,7 +62,7 @@ class MGLIntegrator {
     }
 
     addLayer(layer, beforeLayerID) {
-        const callbackID = `_cartoGL_${uid++}`;
+        const callbackID = `_cartovl_${uid++}`;
```

Comment:
So, according to you @davidmanzanares this id was introduced to be able to remove layers which isn't yet implemented... I guess it's nice to use vl in those internal ids as well anyway